nix/network-docs.nix

```diff
@@ -1,34 +1,25 @@
-{ pkgs }: with pkgs;
+{ pkgs }:
+with pkgs;
 let
   src = haskell-nix.haskellLib.cleanGit {
     name = "ouroboros-network-docs-src";
     src = ../.;
     subDir = "docs";
   };
   cddl-specs = ../ouroboros-network/test-cddl/specs;
-in
-pkgs.runCommand "ouroboros-network-docs"
-{
+in pkgs.runCommand "ouroboros-network-docs" {
   meta.platforms = with pkgs.lib.platforms; [ linux darwin ];
   nativeBuildInputs = [ imagemagick ];
   buildInputs = [
     (texlive.combine {
       inherit (texlive)
-        cleveref
-        framed
-        scheme-small
-        collection-fontsrecommended
-        stmaryrd kpfonts geometry hyperref
-        todonotes
-        amsmath mathtools
-        colortbl polytable lazylist
-        fancyvrb
+        cleveref framed scheme-small collection-fontsrecommended stmaryrd
```

Comment:
Personally, I find it more readable to have one package per-line. Also, it makes it easy to comment out something when tinkering with a nix setup.